tests/onnx/test_onnx_graph_transformations.py

```diff
@@ -58,5 +62,24 @@ def test_weight_sharing_output_match(self):
             )
 
 
+class OnnxMergingTestCase(TestCase):
+    SUPPORTED_ARCHITECTURES_WITH_MODEL_ID = {
+        "gpt2": "optimum/gpt2",
```

Comment:
I would use tiny models here, and use models exported using `optimum.exporters.onnx` (for example this test did not catch the problem in https://github.com/huggingface/optimum/pull/601 )

Reply:
Do you mean that we should have assets for tiny models exported with `optimum.exporters.onnx`?

Reply:
No, I mean using models exported on the fly during the test `optimum.exporters.onnx`.